registry/src/pages/user.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { MDBIcon } from "mdbreact";
+import { useNavigate, useParams } from "react-router-dom";
+import Container from "react-bootstrap/Container";
+import Card from "react-bootstrap/Card";
+import Figure from "react-bootstrap/Figure";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import "./upload.css";
+
+const UserPage = () => {
+  const { user } = useParams();
+  const navigate = useNavigate();
+  const [email, setEmail] = React.useState("");
+  const [dateJoined, setDateJoined] = React.useState("");
+  const [projects, setProjects] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  const url = `${process.env.REACT_APP_REGISTRY_API_URL}/users/${user}`;
```

Comment:
Can you create a separate action for this inside `userActions.js` ? 

And looking at the code, this is called at first render you can use `useEffect` to call the action and pass on `isLoading` in the dependencies array.

Reply:
sure.